_docs-sources/pipelines/tutorial/index.md

```diff
@@ -1,3 +1,159 @@
+
+:::warning
+For a simple proof of concept, the default repo configuration will suffice. Before using these repositories in a production environment, we recommend setting up a [branch protection rule](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule) for your `main` branch. At a minimum, we recommend enabling requiring a pull request before merging with at least one reviewer required.
+:::
+
+### Setting up secrets
+
+Next, configure the required secrets for each repository to allow cross repository access from the `infrastructure-live` and `infrastructure-pipelines` repositories. This is required so `infrastructure-live` can kick off workflows in the `infrastructure-pipeline` repository and the `infrastructure-pipelines` repository can clone your `infrastructure-live` repository as well as access Gruntwork modules.
+
+First, navigate to the `infrastructure-live` repository. Select the `Settings` tab, select the `Secrets and variables` drop down on the left side panel, then select `Actions`. Create three secrets named `GRUNTWORK_CODE_ACCESS_TOKEN`, `PIPELINES_DISPATCH_TOKEN`, and `INFRA_LIVE_ACCESS_TOKEN`. Use your GitHub PAT as the value for all three secrets.
+
+Next, Navigate to the `infrastructure-pipelines` repository. Select the `Settings` tab, select the `Secrets and variables` drop down on the left side panel, then select `Actions`. Create two secrets named `INFRA_LIVE_ACCESS_TOKEN` and `GRUNTWORK_CODE_ACCESS_TOKEN`. Use your GitHub PAT as the value for both secrets.
+
+:::warning
+Using a single token with broad access is sufficient for a POC or demo environments. In a production deployment, we recommend using a mix of fine grained and classic PATS to apply to [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege) to all tokens used in pipelines workflows.
+:::
+
+## Generating code
+
+Next, you’ll generate the IaC and GitHub Actions Workflow code required to run Gruntwork Pipelines. We'll provision an AWS IAM role (which pipelines assumes to create resources), one workflow for the `infrastructure-live` repository, and one workflow for the `infrastructure-pipelines` repository. The template for `infrastructure-live` sets up a folder structure that complies with Gruntwork’s recommended folder structure for Terragrunt, so you could continue to use the generated code in perpetuity and simply add more accounts and resource definitions if you so choose!
+
+### Infrastructure pipelines
+
+First, generate the `infrastructure-pipelines` repository code using Boilerplate. Use the following command replacing `<your GitHub organization name>` with the name of your GitHub organization.
+
+```bash
+boilerplate --template-url "git@github.com:gruntwork-io/terraform-aws-architecture-catalog.git//blueprints/components/infrastructure-pipelines?ref=devops-foundations" \
```

Comment:
Observation: now that we have docs referencing boilerplate templates we should start to think about merging the `devops-foundations` branch and versioning the architecture-catalog repo.

Reply:
I was thinking the same thing